pkg/grpc/universalapi/api_workflow.go

```diff
@@ -269,6 +269,41 @@ func (a *UniversalAPI) PurgeWorkflowBeta1(ctx context.Context, in *runtimev1pb.P
 	return emptyResponse, nil
 }
 
+// GetWorkflowBeta1 is the API handler for getting workflow details
+func (a *UniversalAPI) GetWorkflowBeta1(ctx context.Context, in *runtimev1pb.GetWorkflowRequest) (*runtimev1pb.GetWorkflowResponse, error) {
+	return a.GetWorkflow(ctx, in)
+}
+
+// StartWorkflowBeta1 is the API handler for starting a workflow
+func (a *UniversalAPI) StartWorkflowBeta1(ctx context.Context, in *runtimev1pb.StartWorkflowRequest) (*runtimev1pb.StartWorkflowResponse, error) {
+	return a.StartWorkflow(ctx, in)
+}
+
+// TerminateWorkflowBeta1 is the API handler for terminating a workflow
+func (a *UniversalAPI) TerminateWorkflowBeta1(ctx context.Context, in *runtimev1pb.TerminateWorkflowRequest) (*emptypb.Empty, error) {
+	return a.TerminateWorkflow(ctx, in)
+}
+
+// RaiseEventWorkflowBeta1 is the API handler for raising an event to a workflow
+func (a *UniversalAPI) RaiseEventWorkflowBeta1(ctx context.Context, in *runtimev1pb.RaiseEventWorkflowRequest) (*emptypb.Empty, error) {
+	return a.RaiseEventWorkflow(ctx, in)
+}
+
+// PauseWorkflowBeta1 is the API handler for pausing a workflow
+func (a *UniversalAPI) PauseWorkflowBeta1(ctx context.Context, in *runtimev1pb.PauseWorkflowRequest) (*emptypb.Empty, error) {
+	return a.PauseWorkflow(ctx, in)
+}
+
+// ResumeWorkflowBeta1 is the API handler for resuming a workflow
+func (a *UniversalAPI) ResumeWorkflowBeta1(ctx context.Context, in *runtimev1pb.ResumeWorkflowRequest) (*emptypb.Empty, error) {
+	return a.ResumeWorkflow(ctx, in)
+}
+
+// PurgeWorkflowBeta1 is the API handler for purging a workflow
+func (a *UniversalAPI) PurgeWorkflowBeta1(ctx context.Context, in *runtimev1pb.PurgeWorkflowRequest) (*emptypb.Empty, error) {
+	return a.PurgeWorkflow(ctx, in)
+}
+
 // GetWorkflowAlpha1 is the API handler for getting workflow details
 func (a *UniversalAPI) GetWorkflowAlpha1(ctx context.Context, in *runtimev1pb.GetWorkflowRequest) (*runtimev1pb.GetWorkflowResponse, error) {
```

Comment:
Should Alpha1 methods be removed in this PR? They are aliases for now

Reply:
I'll take advice on this from anyone - not sure of the timelines etc for deprecating this.

Reply:
Not sure either, but wanted to bring that up